src/from/fromMulChannelCap.ts

```diff
@@ -26,7 +26,7 @@ function keyMapper(keys: string[]) {
   });
 }
 
-export function fromMulChannelCap(text: string) {
+export function fromMulChannelCap(text: string, analysis?: Analysis) {
```

Comment:
The second argument could be options={} and one of the options (the only one currently) is analysis.

Reply:
what other option could be added?

Reply:
Currently none. But by habit everything that is optional we put as a property of an option object. It also allows to add more options without breaking changes.